gulpfile.js

```diff
@@ -277,15 +277,15 @@ gulp.task('templatecache', function () {
 
 // Mocha tests task
 gulp.task('mocha', function (done) {
-  // Open mongoose connections
-  var mongoose = require('./config/lib/mongoose.js');
+  var mongooseService = require('./config/lib/mongoose');
```

Comment:
Is there any reason why this `require` is inside the task and not on top of the gulp file with all the other dependencies?